README.md

```diff
@@ -110,13 +109,24 @@ kubectl apply -f https://get.liqo.io/app.yaml -n liqo-demo
 You can observe that:
 
 * Your application is correctly working by exposing the application frontend port and later connecting with a browser to [localhost:8000](localhost:8000). To expose the pod port:
+
 ```bash
   kubectl port-forward -n liqo-demo service/frontend 8080:80
 ```
+
 * Your application is transparently deployed across two different clusters:
+
 ```bash
   kubectl get pods -n liqo-demo -o wide  
-``` 
+```
+
+### Going Further
+
+If you want to understand and get more details, including how to inspect and interact with a service deployed with Liqo, you can explore the documentation website:
+
+* Continue the Liqo journey by exploring the [Liqo playground](https://doc.liqo.io/user/gettingstarted/play/)
+* Find out how to install Liqo on [your clusters](https://doc.liqo.io/user/install/)
+* Read more about the [Liqo architecture](https://doc.liqo.io/architecture/)
```

Comment:
It seems to me the architecture section is mostly WIP. Is it worth adding the link here?